### PR TITLE
Add public changelog, update readme.

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,16 @@ for your car. In doing so, the Vision SDK enables the following user-facing feat
...
 
-[VisionSDK](https://github.com/mapbox/mapbox-vision-android) is a framework written in native language (Kotlin for Android, Swift for iOS) that encapsulates core utilization and platform-dependent tasks. It calls VisionCore.
+[Vision Ar](https://github.com/mapbox/mapbox-vision-android/tree/dev/MapboxVisionAR) is an add-on module for Vision used to create customizable augmented reality experiences. 
+It allows configuration of the user’s route visualization: lane material (shaders, textures), lane geometry, occlusion, custom objects, and more. 
 
-[VisionAR](https://github.com/mapbox/mapbox-vision-ar-android) is a native framework with dependency on the Mapbox Navigation SDK. It takes information from the specified navigation route, transfers it to VisionCore via VisionSDK, receives instructions on displaying the route, and then finally renders it on top of camera frame using native instruments.
+[Vision Safety](https://github.com/mapbox/mapbox-vision-android/tree/dev/MapboxVisionSafety) is an add-on module for Vision used to create customizable alerts for speeding, nearby vehicles, cyclists, pedestrians, lane departures, and more.
```
Comment (RingerJK) [inline README.md]:
```suggestion
[Vision Safety](https://github.com/mapbox/mapbox-vision-android/tree/dev/MapboxVisionSafety) is an add-on module for Vision is used to create customizable alerts for speeding, nearby vehicles, cyclists, pedestrians, lane departures, and more. 
```

Comment (RingerJK) [reply README.md]: I have to do it))

Comment (yunikkk) [reply README.md]: Not sure the change is right, `Vision Safety (...) is and add-on ... used to ...`, seems right to me.

Comment (yunikkk) [reply README.md]: Also I've just copied that from https://docs.mapbox.com/android/vision/overview/

Comment (yunikkk) [reply README.md]: > I have to do it))
Uhhm?

Comment (RingerJK) [reply README.md]: lgtm as well

**Pull request merged**